twistedchecker/test/test_limitedreporter.py

```diff
@@ -1,5 +1,9 @@
 import os
-import StringIO
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
Why does this need to be conditional?  Can't it be `io.StringIO` on both versions?